### PR TITLE
SUMO-231087:Alert Grouping Support for Muting schedules:  update TF f…

### PR DESCRIPTION
**Pull request by jxie418**
…or support group key and value

### CODE REVIEW

**Comment by jxie418** [comment]
run the test on local. 
```
--- PASS: TestAccSumologicMutingSchedulesLibraryMutingSchedule_create_with_group (3.69s)
PASS
ok  	github.com/SumoLogic/terraform-provider-sumologic/sumologic	4.195s
```

File: CHANGELOG.md
Comment (sumovishal) [inline CHANGELOG.md:6]: 2.28.0 is already out. This release note should go under "unreleased" section.

File: website/docs/r/muting_schedule.html.markdown
Comment (sumovishal) [inline website/docs/r/muting_schedule.html.markdown:106]: typo: "Optional"

File: website/docs/r/muting_schedule.html.markdown
Comment (sumovishal) [inline website/docs/r/muting_schedule.html.markdown:124]: Please add an example that demonstrates usage of `notification_groups`.

**Review by sumovishal — COMMENTED** [review]
left some comments

**Comment by sumovishal** [comment]
We can sync up later to look at the failing test.

File: CHANGELOG.md
Comment (jxie418) [reply CHANGELOG.md]: updated.

File: website/docs/r/muting_schedule.html.markdown
Comment (jxie418) [reply website/docs/r/muting_schedule.html.markdown]: Updated.

File: website/docs/r/muting_schedule.html.markdown
Comment (jxie418) [reply website/docs/r/muting_schedule.html.markdown:149]: added.

**Review by sumovishal — COMMENTED** [review]
lgtm, let's wait for the changes to be available everywhere before we merge this PR.

File: website/docs/r/muting_schedule.html.markdown
Comment (gchairuangsang) [reply website/docs/r/muting_schedule.html.markdown:149]: Since we don't have the concept of types in notification_group, `notification_group_type` sounds a little misleading? Can we get rid of the `_type` suffix?

File: website/docs/r/muting_schedule.html.markdown
Comment (gchairuangsang) [inline website/docs/r/muting_schedule.html.markdown:133]: nit: same comment as https://github.com/SumoLogic/terraform-provider-sumologic/pull/601/files#r1450949779

Comment (sumovishal) [reply website/docs/r/muting_schedule.html.markdown:133]: Hmm, I notice the same thing in other places like `monitor_scope_type`. We should fix them all.

File: website/docs/r/muting_schedule.html.markdown
Comment (gchairuangsang) [inline website/docs/r/muting_schedule.html.markdown:105]: nit: formatting

File: website/docs/r/muting_schedule.html.markdown
Comment (gchairuangsang) [inline website/docs/r/muting_schedule.html.markdown:99]: Please change the description to reflect the configuration. The schedule has rrule and monitor ID defined.

File: sumologic/resource_sumologic_muting_schedules_library_muting_schedule_test.go
Comment (gchairuangsang) [inline sumologic/resource_sumologic_muting_schedules_library_muting_schedule_test.go:361]: nit: formatting

File: website/docs/r/muting_schedule.html.markdown
Comment (jxie418) [reply website/docs/r/muting_schedule.html.markdown:149]: OK. let me remove the suffix.

File: website/docs/r/muting_schedule.html.markdown
Comment (jxie418) [reply website/docs/r/muting_schedule.html.markdown:133]: updated.

File: website/docs/r/muting_schedule.html.markdown
Comment (jxie418) [reply website/docs/r/muting_schedule.html.markdown:105]: updated

File: website/docs/r/muting_schedule.html.markdown
Comment (jxie418) [reply website/docs/r/muting_schedule.html.markdown:99]: Updated.

File: sumologic/resource_sumologic_muting_schedules_library_muting_schedule_test.go
Comment (jxie418) [reply sumologic/resource_sumologic_muting_schedules_library_muting_schedule_test.go:361]: It is formatted in the IntelliJ.

File: website/docs/r/muting_schedule.html.markdown
Comment (sumovishal) [inline website/docs/r/muting_schedule.html.markdown:114]: Having these examples can be helpful. Any reason to remove them?

**Review by sumovishal — APPROVED** [review]
LGTM except one minor comment.

File: website/docs/r/muting_schedule.html.markdown
Comment (gchairuangsang) [reply website/docs/r/muting_schedule.html.markdown:114]: I think this seems redundant given that we already provided the link (https://freetools.textmagic.com/rrule-generator) and we already have some examples on the same page above e.g. https://github.com/SumoLogic/terraform-provider-sumologic/pull/601/files#diff-647440352a7c9c50a68f41c831b6e35221582ea5429f951464e1ac25e925fadeR70 .

**Pull request merged**
